### PR TITLE
Form branch

### PR DESCRIPTION
**Pull request by rbreva**
### Project 3: Contact form (pair programming)

Changes required in the specifications have been added.

Thank you for your advices. 🙃

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -322,3 +322,57 @@ header .menu img {
...
+  margin-bottom: 100px;
+}
+
+.form .form_p {
+  font-size: 32px;
+  margin-top: 100px;
+  color: #172b4d;
+  font-weight: bold;
+  margin-bottom: 60px;
+}
```
Comment (franclobo) [inline css/main.css]: - [ ] Kindly add `text-align: center;` to stick to the design as much as possible. 😎 

- Figma template

text-figma

- Your design

contact-text

File: index.html [hunk 2]
```diff
@@ -128,5 +130,23 @@ <h4>About me</h4>
...
+    </form>
+    
+</div>
+<ul class="links">
+    <li class="link"><a href="#"><img src="images/links/github_b.svg" alt="github"></a></li>
+    <li class="link"><a href="#"><img src="images/links/linkedin_b.svg" alt="linkedin"></a></li>
+    <li class="link"><a href="#"><img src="images/links/angelist_b.svg" alt="angelist"></a></li>
+    <li class="link"><a href="#"><img src="images/links/twitter_b.svg" alt="twitter"></a></li>
+    <li class="link"><a href="#"><img src="images/links/medium_b.svg" alt="medium"></a></li>
+</ul>
```
Comment (franclobo) [inline index.html:151]: - [ ] Kindly insert the elements on the bottom of the web page inside the `<footer>` tag, for best practices for HTML & CSS.
- [ ] Please, 🙏🏽 add the line at the end of the web page to be similar to the Figma template as much as possible.  👇🏽 
- [ ] **[OPTIONAL]** Kindly replace the name in the logo "John Doe" with your name. 👍🏽 

- Figma template

line

- Your design

no-line

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @rbreva and @OsvaldoBC , 

Good job so far!
You did it well, keep it on!! 👏🏽 👏🏽 👏🏽 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️
- Correct Github flow ✔️
- Professional README file ✔️
- Descriptive PR ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please, implement the required changes described bellow 👇🏽 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @rbreva \, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**